tests/test_login.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import time
+import requests
+import json
+
+from tests import conftest
+from pages.rptest import RpTest
+
+
+class TestLogin:
+
+    @pytest.mark.nondestructive
+    def test_login(self, selenium, ldap, token):
+        rpTest = RpTest(selenium)
+        rpTest.login_with_ldap(ldap['email'], ldap['password'], conftest.passcode(ldap['secret_seed']))
+        access_token = conftest.access_token(token)
+        api = "https://auth-dev.mozilla.auth0.com/api/v2/users/ad%7CMozilla-LDAP-Dev%7Cparsys_test/logs"
+        user_logs = requests.get(api, headers={'Authorization': 'Bearer {0}'.format(access_token)})
+        initial_user_log_date = json.loads(user_logs.content)[0]['date']
+        time.sleep(16)
```

Comment:
Isn't there a more deterministic way instead of waiting until ``time.sleep(16)``? Also why ``16`` :P ?